worlds/noita/Locations.py

```diff
@@ -118,71 +119,86 @@ class HiddenChests(IntEnum):
     },
     "Overgrown Cavern": {
         "Overgrown Cavern Chest": LocationData(112200, "hc", HiddenChests.main_world),
+        "Overgrown Cavern Pedestal": LocationData(112700, "peds", Pedestals.main_world),
```

Comment:
We'll need ranges otherwise there is only 1 chest and 1 pedestal

Reply:
I'm doing the numbered chests and pedestals in Regions instead -- it made the most sense to me to do it there, since we need to use the slider to determine the number of checks needed, and we don't currently have access to that value in Locations.

Reply:
Ah ok I see.